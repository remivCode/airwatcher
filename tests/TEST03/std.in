1
user
pass
1
1
Sensor1
1
Sensor2
1
Sensor3
0
0