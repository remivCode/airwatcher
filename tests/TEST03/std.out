TEST EXECUTION
----- AUTHENTIFICATION PHASE -----

0. End
1. Authentification

If you already have an account, please enter your username and your password:
Username: 
Password: 

Enter the user (1: Government, 2: Private, 3: Provider)
Enter a number:
----- Government Agency Menu -----
1: Analyze Sensors data
2: Private Sensors data
3: Sensors similarity
4: Sensors statistics
5: Air quality
6: Air Cleaners impact
0: End
Enter your choice:

Enter the sensor ID
Enter une chaine de caractère:
The sensor is perfectly working.
----- Government Agency Menu -----
1: Analyze Sensors data
2: Private Sensors data
3: Sensors similarity
4: Sensors statistics
5: Air quality
6: Air Cleaners impact
0: End
Enter your choice:

Enter the sensor ID
Enter une chaine de caractère:
The sensor is perfectly working.
----- Government Agency Menu -----
1: Analyze Sensors data
2: Private Sensors data
3: Sensors similarity
4: Sensors statistics
5: Air quality
6: Air Cleaners impact
0: End
Enter your choice:

Enter the sensor ID
Enter une chaine de caractère:
The sensor is not functional
----- Government Agency Menu -----
1: Analyze Sensors data
2: Private Sensors data
3: Sensors similarity
4: Sensors statistics
5: Air quality
6: Air Cleaners impact
0: End
Enter your choice:

----- AUTHENTIFICATION PHASE -----

0. End
1. Authentification